go.mod

```diff
@@ -11,20 +11,17 @@ require (
 	github.com/decred/dcrd/chaincfg v1.3.0
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
 	github.com/decred/dcrd/dcrec/secp256k1 v1.0.1
-	github.com/decred/dcrd/dcrutil v1.2.0
+	github.com/decred/dcrd/dcrutil v1.2.1-0.20190118223730-3a5281156b73
 	github.com/decred/dcrd/hdkeychain v1.1.1
 	github.com/decred/dcrd/wire v1.2.0
-	github.com/decred/dcrdata/v3 v3.1.2
+	github.com/decred/dcrdata/api/types v1.0.3-0.20190325154125-93d91434f922
```

Comment:
One commit past the tagged version since the go.sum was generated after the tag (although it was a module with a go.mod on the tag).